src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const threeDayRent = 20;
+  const sevenDayRent = 50;
+  const rentSum = days * dayRent;
+
+  if (days >= 7) {
+    return rentSum - sevenDayRent;
+  }
+
+  if (days >= 3) {
```

Comment:
```suggestion
  if (days >= 7) {
    return rentSum - sevenDayRent;
  } else if (days >= 3) {
```
Use `if - else if` construction

Reply:
окей